apps/client/src/components/molecules/button/Button.component.tsx

```diff
@@ -1,14 +1,16 @@
+import cs from 'classnames';
 import { ReactElement } from 'react';
 import { Text } from '../../';
 import { ReactComponent as CarIcon } from '../../../assets/cars/pixelCar.svg';
-import cs from 'classnames';
 
 export interface ButtonProps {
-  onClick: () => void;
+  onClick: (e: React.MouseEvent<HTMLButtonElement>) => void;
   isCarVisible?: boolean;
   isDisable?: boolean;
   isButtonDanger?: boolean;
   children: string;
+  input?: ReactElement;
+  isFillWidth?: boolean;
```

Comment:
`isFullWidth`

Reply:
Fixed in 9285214b129594bafcc9e8e797229d82d65898c7 (PR #52)